packages/ui/primitives/react-hooks/src/useAsyncCallback.ts

```diff
@@ -4,6 +4,9 @@
 
 import { useEffect, useState } from 'react';
 
+/**
+ * @deprecated
```

Comment:
why?

Reply:
I was planning to merge with useAsyncEffect but decided against pending a longer discussion.